examples/create_deployment.py

```diff
@@ -27,7 +27,7 @@ def main():
 
     with open(path.join(path.dirname(__file__), "nginx-deployment.yaml")) as f:
         dep = yaml.safe_load(f)
-        k8s_beta = client.ExtensionsV1beta1Api()
+        k8s_beta = client.AppsV1Api()
```

Comment:
please rename `k8s_beta` to something that makes more sense, e.g. `apps_v1`. Otherwise lgtm